lib/services/generic_resource.rb

```diff
@@ -63,7 +63,7 @@ def self.default_http_options
   end
 
   def allocate_logger
-    @logger = Logger.new(STDOUT)
+    @logger = AhaLogger.new(STDOUT)
```

Comment:
@alexbartlow do you know if this would cause any problems w/ live? I think it was accidently not using our subclassed logger, which added a ton of noise to our specs.